server/tests/server_harness/mod.rs

```diff
@@ -101,8 +101,8 @@ where
             .read_line(&mut output)
             .expect("could not read stdout");
         if !output.is_empty() {
-            stdout().write(b"SRV: ").unwrap();
-            stdout().write(output.as_bytes()).unwrap();
+            stdout().write_all(b"SRV: ").unwrap();
```

Comment:
luckily this is only test code, these look like they could have been actual bugs!